config/insultibot/SampleUtterances.txt

```diff
@@ -4,4 +4,18 @@ INSULT to give me an insult
 BACKHANDED_COMPLIMENT to compliment me
 BACKHANDED_COMPLIMENT for a compliment
 BACKHANDED_COMPLIMENT to give me a compliment
-HELP for a little bit of help
\ No newline at end of file
+HELP for a little bit of help
+HELP how do I use you
+HELP how do I use it
+WHO_BUILT_YOU who built you
+WHO_BUILT_YOU who built it
+WHO_BUILT_YOU how it was built
+WHAT_DO_YOU_DO what does it do
+WHAT_DO_YOU_DO what do you do
+FRIENDS who are your friends
+FRIENDS who are its friends
+FRIENDS who its friends are
+WHO_IS who is {botName}
+WHO_IS who {botName} is
```

Comment:
WHO_IS to tell me who {botName} is
WHO_IS to tell us who {botName} is
WHO_IS to explain who {botName} is
WHO_IS to explain to me who {botName} is
WHO_IS to explain to us who {botName} is